API.md

```diff
@@ -342,6 +342,7 @@ Get all available addons.
             "url": "null|url",
             "logo": "bool",
             "audio": "bool",
+            "audio": "bool",
```

Comment:
Shouldn't this here be `"gpio": "bool"`?

Reply:
Yes I fix it. But that is only inside description of API